dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/ObjectCreationNode.java

```diff
@@ -132,7 +148,7 @@ public String toString() {
         if (enclosingExpression != null) {
             sb.append(enclosingExpression + ".");
         }
-        sb.append("new " + constructor + "(");
+        sb.append("new " + identifier + "(");
```

Comment:
This output is also missing constructor type arguments. For consistency, should this go to the tree and see whether there are constructor type arguments and output them in the correct location?

Reply:
How about `return getTree().toString()`? I compared this way and found the output is pretty good.